vector/src/main/java/im/vector/app/features/VectorFeatures.kt

```diff
@@ -36,4 +38,5 @@ interface VectorFeatures {
 
 class DefaultVectorFeatures : VectorFeatures {
     override fun onboardingVariant(): VectorFeatures.OnboardingVariant = BuildConfig.ONBOARDING_VARIANT
+    override fun isAlreadyHaveAccountSplashEnabled() = true
```

Comment:
the `I already have an account` is enabled by default when the onboarding variant is set to  `FTUE_AUTH`, it's currently set to `LEGACY`, will raise a dedicated PR to enable the `FTUE_AUTH` onboarding as the default variant